internal/zinc-scripted/src/main/contraband-scala/sbt/internal/inctest/Project.scala

```diff
@@ -7,22 +7,23 @@ package sbt.internal.inctest
 final class Project private (
   val name: String,
   val dependsOn: Vector[String],
-  val in: Option[java.io.File]) extends Serializable {
+  val in: Option[java.io.File],
+  val scalaVersion: Option[String]) extends Serializable {
   
-  private def this(name: String) = this(name, Vector(), None)
+  private def this(name: String) = this(name, Vector(), None, None)
```

Comment:
If file is generated then it should be included in .gitingnore (probably whole `contraband-scala` dir). Do I miss something here?